src/main/java/hubturbo/embeddedbrowser/jxbrowser/JxBrowserAdapter.java

```diff
@@ -105,7 +105,7 @@ public void reset() {
         this.browserView.getBrowser()
                         .getLoadListeners()
                         .stream().forEach(listener -> browserView.getBrowser().removeLoadListener(listener));
-        this.browserView.sceneProperty().removeListener(sceneChangeListener);
+        this.attachListener = Optional.empty();
```

Comment:
The previous is a mistake. attachListener is the listener that new page passed in.